tests/openssl_moretest/openssl_moretest.edl

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    from "openenclave/edl/fcntl.edl" import *;
+#ifdef OE_SGX
+    from "openenclave/edl/sgx/platform.edl" import *;
+#else
+    from "openenclave/edl/optee/platform.edl" import *;
+#endif
+
+    trusted {
+        public void ecall_set_rdrand_engine(void);
+
+        public int ecall_schema_run_digest_tests([user_check]void* buf);
+        //public int ecall_asym_tests([user_check]void* buf);
```

Comment:
Can you add comments explain that the following ecall are TODO, or remove them and add later?

Reply:
Remove commented out code

Reply:
I have another fork https://github.com/jinghe-INTC/openenclave/tree/add_openssl_moretest for the rest of the test, to be submitted in another PR when this one is done. I can remove the commented lines.